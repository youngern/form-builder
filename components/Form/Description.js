import React from 'react';
import { StyleSheet, Text } from 'react-native';

import Config from 'final-form-react-native/services/config';
const { Fonts } = Config;
const Description = ({ value }) => {
  return <Text style={styles.description}>{value.trim()}</Text>;
};

const styles = StyleSheet.create({
  description: {
    fontFamily: Fonts.regular,
    color: '#679b9b',
    fontSize: 12,
  },
});

export default Description;
