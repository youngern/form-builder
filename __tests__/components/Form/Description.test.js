import React from 'react';
import Description from 'FinalFormReactNative/components/Form/Description';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Description value={'test link'}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
