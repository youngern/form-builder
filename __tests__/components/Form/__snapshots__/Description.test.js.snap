// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#679b9b",
      "fontFamily": undefined,
      "fontSize": 12,
    }
  }
>
  test link
</Text>
`;
